sigopt/xgboost/run.py

```diff
@@ -12,13 +12,14 @@
 from ..log_capture import SystemOutputStreamMonitor
 from .. import create_run
 
-DEFAULT_EVALS_NAME = 'Test Set'
+DEFAULT_EVALS_NAME = 'TestSet'
```

Comment:
Should we also remove the space in `Training Set`

Reply:
good point, done